### PR TITLE
fix: read storage without get

### LINKED ISSUE #623: Support reading storage variable directly without get
**Is your feature request related to a problem? Please describe.**
Instead of `ensemble.storage.get('attr')`, we should be able to reference attr like so `ensemble.storage.attr`

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sagardspeed2**
## Describe your changes
Added support to read storage value without get method

## Issue ticket number and link

Closes #623 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests
- [x] I have added a changeset `pnpm changeset add`
- [x] I have added example usage in the kitchen sink app

### CODE REVIEW

File: packages/framework/src/hooks/useEnsembleStorage.ts [hunk 1]
```diff
@@ -33,12 +37,16 @@ export const useEnsembleStorage = (): EnsembleStorage => {
     [setStorage, storageBuffer],
   );
 
+  useMemo(() => {
```
Comment (evshi) [inline packages/framework/src/hooks/useEnsembleStorage.ts:40]: What is this hook for?

File: packages/framework/src/hooks/useEnsembleStorage.ts [hunk 2]
```diff
@@ -3,26 +3,30 @@ import { createJSONStorage, atomWithStorage } from "jotai/utils";
 import { clone, merge } from "lodash-es";
 import { useMemo } from "react";
 
-export interface EnsembleStorage {
+export type EnsembleStorage = {
```
Comment (evshi) [inline packages/framework/src/hooks/useEnsembleStorage.ts:6]: Let's keep the definition in the type file and import it here

**Comment by sharjeelyunus** [comment]
@sagardspeed2 does this also works for set? 
user should also be able to set the values in storage without using set like this: `ensemble.storage.apidata = response.body`

**Comment by sagardspeed2** [comment]
> ser should also be able to set the values in st

No, this doesn't support in react at this time.

**Review by evshi — APPROVED** [review]

**Comment by evshi** [comment]
@sharjeelyunus made this to track for now https://github.com/EnsembleUI/ensemble-react/issues/635

**Pull request merged**